labels.json

```diff
@@ -391,6 +571,11 @@
                 "lang": "en",
                 "name": "Possum",
                 "description": "This user is a Possum! (Phalangeriformes; NOT the same as an opossum which is sometimes called \"possum\" as well in North America)"
+            },"
```

Comment:
This batch of translations is wonderful, thank you! One minor fix, can you remove this extra double-quote please? After that, I'll approve!